scripts/submit_gaussian.bash

```diff
@@ -6,6 +6,11 @@ WORKING_DIR=$(dirname "$SCRIPTS_DIR")
 INPUT_DIR="$WORKING_DIR/input"
 OUTPUT_DIR="$WORKING_DIR/output"
 LOG_DIR="$WORKING_DIR/logs"
+PARTITION=$1
+
+if [ -z "$PARTITION" ]; then
```

Comment:
This if statement can be replaced with:
`: ${PARTITION:="short"}`

Reply:
what does : ${PARTITION:="short"} mean exactly?

Reply:
Sets a default basically, see [this question](https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts), actually, we can remove line 9 too and simply put:
`PARTITION="${1:="short"}"`